cookiecutter.json

```diff
@@ -9,7 +9,7 @@
     "repo_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
     "project_short_description": "Python package for doing science.",
     "year": "2018",
-    "minimum_supported_python_version": ["3.6", "3.5", "3.4"],
+    "minimum_supported_python_version": ["3.6", "3.7", "3.5", "3.4"],
```

Comment:
including 3.4?

Reply:
Sorry, what’s the question?

Reply:
Do we need to include py3.4 here? I thought 3.5+ are good enough?

Reply:
> Do we need to include py3.4 here?

@licode 3.4 is still the newest version available on some Linux distributions. Although we do recommend people avoid using system Python and instead use conda or pyenv where it's easy to grab 3.7,  it's still reasonable to support 3.4 users at this point.

> I'd also recommend to sort the versions as they were before (in a future PR).

Since we are still pretty early in the lifecycle of 3.7, making it the default _minimum supported version_ seemed to aggressive. You might even argue for making 3.5 or 3.4 the default. I agree that having them sorted sequentially would be better, but cookiecutter requires that the default option be the first one listed, so I think this is the best compromise.